scripts/release.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
`#!/bin/bash` because we use `[[ =~ ]]` below, which is a bash syntax.

Reply:
Updated! thanks!